### PR TITLE
Corrected Webpack configuration in JavaScript documentation

### PR DESCRIPTION
**Pull request by adarshbhat**
When using the JavaScript code generated by antlr4 in a web browser environment, we have to make exceptions for certain node.js based global modules. The configuration provided in documentation is incorrect and no longer works on new versions of Webpack. This PR attempts to correct the documentation with new configuration that works. Thanks to the Webpack team for pointing out the correct approach.

My understanding is that even after switching to ES6, we will still need this configuration.

### CODE REVIEW

File: doc/javascript-target.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ The steps to create your parsing code are the following:
  
 You are now ready to bundle your parsing code as follows:
  - following webpack specs, create a webpack.config file
- - in the `webpack.config` file, exclude node.js only modules using: `node: { module: "empty", net: "empty", fs: "empty" }`
+ - in the `webpack.config` file, exclude node.js only modules using: `resolve: { fallback: { fs: false } }`
```
Comment (ericvergnaud) [inline doc/javascript-target.md:71]: Not everybody is keen to move to latest webpack, so suggest we say l71 is for webpack vn whilst l72 is for webpack vn+1

Comment (adarshbhat) [reply doc/javascript-target.md:71]: Let me check whether or not the configuration works in the previous version of Webpack. Based on other people's conversations I found on Google, it doesn't work there either. It probably worked 2 major versions ago. But let me confirm.

Comment (adarshbhat) [reply doc/javascript-target.md:71]: You are right. Someone else confirmed that the configuration fails only in the latest version of webpack. I will update this PR to reflect that.

**Comment by ericvergnaud** [comment]
@parrt blessed

**Pull request merged**